test/box-luatest/gh_8192_feedback_daemon_metrics_test.lua

```diff
@@ -28,6 +28,8 @@ end)
 
 g.before_each(function()
     g.server:exec(function()
+        -- Replace module metrics with empty table to avoid metrics collection.
+        package.loaded["metrics"] = {}
```

Comment:
Why not simply `nil`?

Reply:
But you don't call `require` here. And if it's required internally by `box.cfg`, then I don't understand how it works with an empty table:
https://github.com/tarantool/tarantool/blob/606e50c4d56bb76714dc41339102b32230fd751e/src/box/lua/load_cfg.lua#L529

Reply:
I tried to change it to `nil` and run the test — everything works fine.